docs/cip/CIP-12062023_System_Info.md

```diff
@@ -0,0 +1,214 @@
+# CIP-7: System Info
+
+## Status
+
+Current Status: `Under Discussion`
+
+## Motivation
+
+Currently, a lot of the support discussions in Discord revolve around gathering information about the user's operating
+environment. This information is crucial for debugging and troubleshooting issues. We want to make it easier for users
+to provide this information.
+
+## Public Interfaces
+
+This proposal introduces a new `API` endpoint `/api/v1/env` that will return a dictionary with system information based
+on flags provided by the user.
+
+The endpoint will accept the following flags:
+
+- `python_version` - Shows Python related information. Possible values `True` or `False` (default `True`)
+- `os_info` - Shows OS related information. Possible values `True` or `False` (default `True`)
+- `memory_info` - Shows memory related information. Possible values `True` or `False` (default `True`)
+- `cpu_info` - Shows CPU related information. Possible values `True` or `False` (default `True`)
+- `disk_info` - Shows disk related information. Possible values `True` or `False` (default `False`)
+
+By default, the endpoint will display the following information (example below):
+
+- `chroma_version` - The version of Chroma
+- `chroma_settings` - The settings used to start Chroma
+- `datetime` - The current date and time
+- `persist_directory` - The directory where Chroma is storing its data
+
+For security reasons the endpoint will be disabled by default. It can be enabled by setting
+the `CHROMA_SERVER_ENV_ENDPOINT_ENABLED=1` on the server.
```

Comment:
👍 good call

Reply:
With @HammadB, we agreed always to expose it but protected it with auth. So I need to remove `CHROMA_SERVER_ENV_ENDPOINT_ENABLED` as it is no longer used.

We also agreed to remove the flags; I haven't updated the CIP doc for that.